test/functional/update/delta-without-rename-flag/test.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+setup() {
+  clean_state_dir
+  clean_target_dir
+  create_manifest_tar 10 MoM
+  sign_manifest_mom 10
+  create_manifest_tar 10 os-core
+  create_manifest_tar 20 MoM
+  sign_manifest_mom 20
+  create_manifest_tar 20 os-core
+  set_os_release 10
+  chown_root -R "$DIR/target-dir" "$DIR/web-dir"
+  unpack_target_from_manifest 10 MoM
+  unpack_target_from_manifest 10 os-core
+  sudo tar -C "$DIR/web-dir/20" -cf "$DIR/web-dir/20/pack-os-core-from-10.tar" delta/ staged/
+  echo "CMD: $SWUPD update $SWUPD_OPTS"
+}
+
+teardown() {
+  clean_tars 10
+  clean_tars 20
+  revert_chown_root -R "$DIR/web-dir"
```

Comment:
You will want to add `"$DIR/target-dir"` as an additional argument to this line, since it is chowned to root in setup().

Reply:
Done.